templates/docs/examples/patterns/card/_header-and-footer.html

```diff
@@ -0,0 +1,9 @@
+<div class="p-card">
+  <img class="p-card__thumbnail" src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,c_fill,w_64,h_64/https://api.charmhub.io/api/v1/media/download/charm_B1WVq07SAhAIBJ72odJQ9fNBCFXk7QYT_icon__ce0ecd4c4fb29e9a8ff84971fbbba8552a822d53717632f2aa5ecbce9dfea108.png" alt="landscape-server" class="p-card__thumbnail" loading="lazy" width="32" height="32" />
```

Comment:
I feel like I commented on this before - please change the image to something from asset server, not claudinary.

Reply:
fixed